src/libraries/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -348,8 +356,16 @@ public static IEnumerable<object[]> ToUpper_TestData()
                 yield return new object[] { cultureName, "\u0149", "\u0149" };
             }
 
+            var locales = new List<string> { "tr", "tr-TR"};
+            if (PlatformDetection.IsNotBrowser)
+            {
+                // Browser's ICU doesn't contain these locales
```

Comment:
Maybe move to a shared property to avoid duplication? Something like, `GetTestLocales`?